mungegithub/mungers/close-stale-pr.go

```diff
@@ -222,7 +222,7 @@ func durationToDays(duration time.Duration) string {
 }
 
 func closePullRequest(obj *github.MungeObject, inactiveFor time.Duration) {
-	mention := mentionUsers(getInvolvedUsers(obj))
+	mention := mungerutil.JoinMentions(mungerutil.CreateActiveUsersFromIssue(obj.Issue).AllUsers())
```

Comment:
Would `GetActiveUsersFromIssue` be a better name? I found line 225 a little confusing when reading it. Although mungetutil is creating an object internally, because the function is exported, `GetSomething` may be more meaningful as a name.

I feel like mungerutil should export functions like GetUsersFromIssue, GetAssigneesFromIssue, and GetAuthorsFromIssue, which would make using it easier. Does that make sense?